pombola/kenya/static/js/survey.js

```diff
@@ -0,0 +1,75 @@
+(function() {
```

Comment:
Since this is a fairly long function, I think it'd be helpful to make the local variable names more descriptive than these one or two character names.